kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/SetupMethod.vue

```diff
@@ -41,58 +60,107 @@
 
   import { SelectAddressModalGroup } from 'kolibri.coreVue.componentSets.sync';
   import commonSyncElements from 'kolibri.coreVue.mixins.commonSyncElements';
+  import CoreInfoIcon from 'kolibri.coreVue.components.CoreInfoIcon';
   import OnboardingForm from './OnboardingForm';
 
   export default {
     name: 'SetupMethod',
     components: {
+      CoreInfoIcon,
       OnboardingForm,
       SelectAddressModalGroup,
     },
     mixins: [commonSyncElements],
     data() {
       return {
         showAddressModal: false,
+        showLODAddressModal: false,
+        initializeMethod: 'new',
       };
     },
+    inject: ['wizardService'],
     methods: {
+      handleSubmit() {
+        switch (this.initializeMethod) {
+          case 'new':
+            this.startNewFacilityFlow();
+            break;
+          case 'import':
+            this.showAddressModal = true;
+            break;
+          case 'lod':
+            this.showLODAddressModal = true;
+            break;
+        }
+      },
+
       startNewFacilityFlow() {
-        this.$router.push({ path: '/create_facility/1' });
+        this.wizardService.send({ type: 'CONTINUE', value: 'new' });
       },
       startFacilityImportFlow(address) {
+        this.wizardService.send({ type: 'CONTINUE', value: 'import' });
         this.$router.push({
           path: '/import_facility/1',
           query: {
             device_id: address.id,
           },
         });
       },
+      startLODImportFlow(address) {
+        this.wizardService.send({ type: 'CONTINUE', value: 'lod' });
+        this.$router.push({
+          path: '/import_lod/1',
+          query: {
+            device_id: address.id,
+          },
+        });
+      },
     },
     $trs: {
-      setupMethodHeader: {
-        message: 'Create or import facility',
+      initializeDevice: {
+        message: 'Initialize device',
```

Comment:
We changed the wording for the heading here, [check the latest on Figma](https://www.figma.com/file/w3raYcGRUhhaVm1Of8Dcvo?node-id=229:63#94924868):

![Selection_005](https://user-images.githubusercontent.com/1457929/128079143-3e408d3d-36e7-444e-9cd5-424b5a96485a.png)

Reply:
Thanks, done, updated in the animated gif above.